### PR TITLE
Feature/midl io

### PR DESCRIPTION
**Pull request by pmandler**

### CODE REVIEW

File: venues/MIDL.io/2019/Conference/Full/process/submissionProcess.js [hunk 1]
```diff
@@ -28,6 +28,38 @@ function () {
             readers: invitation.reply.readers
         }
    };
+   var commentInvitation = {
+        id: CONFERENCE_ID + '/-/Paper'+note.number+'/Pams_Comment',
```
Comment (melisabok) [inline venues/MIDL.io/2019/Conference/Full/process/submissionProcess.js]: Pams_Comment?

File: venues/MIDL.io/2019/Conference/Full/python/full.py [hunk 2]
```diff
@@ -131,15 +125,20 @@
...
+        'description': 'Upload a PDF file that ends with .pdf. There is no strict limit on paper length. However, we strongly recommend keeping the paper at 8 pages, plus 1 page for the references and as many pages as needed in an appendix section (all in a single pdf).',
         'order': 9,
         'value-regex': 'upload',
         'required': True
     },
-    'author affiliation': {
-        'description': 'Institution name(s) of the author(s)',
-        'order': 10,
-        'values-regex': "[^;,\\n]+(,[^,\\n]+)*",
+    'conduct of conduct': {
```
Comment (melisabok) [inline venues/MIDL.io/2019/Conference/Full/python/full.py]: code of conduct

File: venues/MIDL.io/midl-landing-webfield.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+var GROUP_ID = 'MIDL.io';
+var HEADER = {
+  title: 'Medical Imaging with Deep Learning',
+  description: ''
+};
+var VENUE_LINKS = [
+  { url: '/group?id=MIDL.io/2019/Conference/Full', name: 'MIDL 2019 Full Paper' },
+  { type: 'divider' },
```
Comment (melisabok) [inline venues/MIDL.io/midl-landing-webfield.js]: I didn't know we had a divider

File: venues/MIDL.io/2019/Conference/Full/process/submissionProcess.js (hunk 1 above)
Comment (pmandler) [reply venues/MIDL.io/2019/Conference/Full/process/submissionProcess.js]: Oops!  I'm still working on the comments as I'm not clear on who should be able to see them.  I'll get that out of there.

**Comment by melisabok** [comment]
Should I merge this?

**Pull request merged**